spyder/app/tour.py

```diff
@@ -28,10 +28,39 @@
 
 # Local imports
 from spyder.config.base import _, get_image_path
+from spyder.config.main import CONF
+from spyder.config.gui import is_dark_font_color
 from spyder.py3compat import to_binary_string
 from spyder.utils.qthelpers import add_actions, create_action
 from spyder.utils import icon_manager as ima
 
+
+def get_colors():

```

Comment:
@dalthviz, What if define a function a global function called `is_dark_interface` that returns True if the interface is currently dark and False otherwise?

The function could be something like:

```python
def is_dark_interface():
    ui_theme = CONF.get('color_schemes', 'ui_theme')
    color_scheme = CONF.get('color_schemes', 'selected')
    if ui_theme == 'dark':
        return True
    elif ui_theme == 'automatic':
        if not is_dark_font_color(color_scheme):
            return True
        else:
            return False
    else:
        return False
```

That function could go in `spyder.config.gui` (like `is_dark_font_color`) and would allow you to remove the `get_colors`, `get_background_color`, etc functions you introduced, to write instead something like

```python
if is_dark_interface():
    MAIN_TOP_COLOR, MAIN_BG_COLOR = QColor.fromRgb(35, 38, 41)
else:
    MAIN_TOP_COLOR = QColor.fromRgb(230, 230, 230)
    MAIN_BG_COLOR = QColor.fromRgb(255, 255, 255)
```